docs/dataset.rst

```diff
@@ -82,3 +82,22 @@ DatasetCifar
 
 .. autoclass:: DatasetCifar
     :members:
+
+
+NLP
+~~~~~~~~~~~~~~~~
+
+.. automodule:: mlbench_core.dataset.nlp
```

Comment:
Use `autoapimodule`, `autoapiclass` and `autoapifunction` so that the docs works without installing mlbench-core